wordpress-mobile/WordPress-Android
# Notifications: Updates noticons font.

Fixes #4586 

To test:
Launch the app and check notifications.

Confirm that layout of noticon labels is correct.
Test a vault press notification and ensure the correct noticon is displayed.

Needs review: @theck13 would you do the honors?
